test/cypress/e2e/home.cy.ts

```diff
@@ -216,6 +216,14 @@ describe('Visits the home page', () => {
                 cy.get('[data-testid="hamburger-menu-footer-copyright"]').should('exist').should('be.visible')
                 cy.get('[data-testid="hamburger-menu-close-button"]').should('exist').click()
             })
+
+            // theme switcher
+            it('theme switcher is visible', () => {
```

Comment:
Nit: The `describe()` and `it()` should basically read like sentences. In this case, you'll want the `it()` to start with a verb.

```suggestion
            it('shows the theme switcher', () => {
```